saucal/woocommerce-gateway-amazon-pay
# Making Plugin compatible with HPOS

### All Submissions:






### Changes proposed in this Pull Request:
Making Amazon Pay plugin compatible with HPOS

Made the plugin compatible using this [guide](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#declaring-extension-incompatibility)

Skipped:
- includes/compats/class-wc-amazon-payments-advanced-multi-currency-abstract.php L215-216



Closes woo#226 .

### How to test the changes in this Pull Request:

1. [Enable](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#how-to-enable-hpos) HPOS in your installation.
2. Plugins functionality should remain the same regardless of HPOS unless you are using legacy API Keys

### Other information:




### Changelog entry

> ADD - Compatibility with HPOS
